global.json

```diff
@@ -1,16 +1,16 @@
 {
   "sdk": {
-    "version": "6.0.100",
+    "version": "7.0.100-preview.4.22204.1",
     "allowPrerelease": true,
     "rollForward": "major"
   },
   "tools": {
-    "dotnet": "6.0.100"
+    "dotnet": "7.0.100-preview.4.22204.1"
```

Comment:
We usually only depend on a signed and publicly available SDK (no nightly build). I assume that one is a nightly build?

Reply:
That's very unfortunate :(

Just for my understanding, why was a P4 SDK chosen and not a P3?